.gitignore

```diff
@@ -4,4 +4,6 @@ node_modules/
 .idea/
 # Sed backup files.
 *-e
+# Vim swap files.
+*.swp
```

Comment:
I use global `~/.gitignore` for all projects so I don't have to add `.gitignore` to each repo I work on. 

To be pedantic I would gladly remove `.gitignore` file from this repo if most people use `core.excludesfile` directive. 

What say you people ?